README.md

```diff
@@ -43,6 +43,12 @@ npm install babel-cli
 > Note for npm 2 users, you have to make sure po2json in the root `node_modules`, the same way as
 `babel-cli`
 
+And finally, install `babel-runtime` in your dev-dependencies needed to be able to compile the code.
```

Comment:
`babel-cli` comes with `babel-runtime` so I didn't got the error.

Well it could be shortened is the previous section:

```
Make sure you have `babel-cli` (used by i18n scripts) and `babel-runtime` (for code compilation) in your dependencies (it's up to you to select the version saved in your project):
```

```
npm install --save-dev babel-cli babel-runtime`
```